### PR TITLE
Nc news task 6

### PR DESCRIPTION
**Pull request by swlho**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -60,6 +60,30 @@ describe("/api/topics", () => {
...
+							author: expect.any(String),
+							title: expect.any(String),
+							article_id: expect.any(Number),
+							topic: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							article_img_url: expect.any(String),
+							comment_count: expect.any(Number),
+						});
+						expect(article).not.toHaveProperty("body");
```
Comment (nc-hannah-burks) [inline __tests__/app.test.js:82]: Nice testing!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -98,31 +122,69 @@ describe("/api/articles", () => {
...
+						votes: expect.any(Number),
+						created_at: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						article_id:expect.any(Number)
+					})
+				})
+			})
+		})
+		test('STATUS 200: If the requested article only has one comment, returns the comment as an object', () => {
```
Comment (nc-hannah-burks) [inline __tests__/app.test.js:145]: Interesting edge case, although I am not sure how much sense it makes returning a single object on a key of 'comments'. If we use an array, it hints to the fact there can be more than 1 comment

File: __tests__/app.test.js [hunk 3]
```diff
@@ -98,31 +122,69 @@ describe("/api/articles", () => {
...
+						article_id: 6,
+						created_at: expect.any(String)
+				})
+			})
+		})
+		test('STATUS 204: If the requested article_id has no associated comments, returns empty array', () => {
+			return request(app)
+			.get("/api/articles/13/comments")
+			.expect(200)
+			.then(({body})=>{
```
Comment (nc-hannah-burks) [inline __tests__/app.test.js:164]: In your description you have wrote status 204, but then expected status 200. Status 204 or 200 would be fine here.

File: __tests__/app.test.js (hunk 2 above)
Comment (nc-hannah-burks) [reply __tests__/app.test.js:145]: You might wish to add a test where you somehow check the comment count is correct.
